src/ensembl/src/content/app/browser/drawer/sampleData.ts

```diff
@@ -0,0 +1,518 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// FIXME: Not sure if we need to use source here. If yes, we need to move the type definition to the shared folder.
+import { Source } from 'src/content/app/entity-viewer/types/source';
+
+export type TrackDetails = {
+  track_id: string;
+  track_name: string;
+  strand: 'forward' | 'reverse' | null;
+  shared_description?: string;
+  specific_description?: string;
+  source?: Source;
+};
```

Comment:
Wonder what determines the colour of the squares. I suppose it's just protein coding vs non-protein-coding. Wonder where that information should go.

As well as where should the information responsible for the name of tracks group (e.g. "Genes & transcripts") go.

![image](https://user-images.githubusercontent.com/6834224/105434386-62873080-5c53-11eb-81fb-9e1a152c48d6.png)

We probably need some kind of collective brainstorming session to discuss what tracks are and what fundamental properties they have. `shared_desciption` and `specific_desription` look suspicious (I understand that you want to separate them into their own paragraphs).